### PR TITLE
Overview

### PR DESCRIPTION
**Pull request by kycodee**

### CODE REVIEW

File: client/components/HateMailComponents/HateMail.jsx [hunk 1]
```diff
@@ -97,7 +117,10 @@ const HateMail = ({ user }) => {
               required
             />
           </div>
-          <button type="submit">Send Hatemail</button>
+          <button type="submit" onClick={() => {
+            postConflict()
+            console.log('src', recipients)
```
Comment (peytono) [inline client/components/HateMailComponents/HateMail.jsx:121]: There's a few leftover logs in this file and look like some unused code between 48-51

**Review by peytono — COMMENTED** [review]
Good work, just seeing leftover logs!

File: client/components/HateMailComponents/HateMail.jsx (hunk 1 above)
Comment (kycodee) [reply client/components/HateMailComponents/HateMail.jsx:121]: Thank you, they've been removed.

**Review by peytono — APPROVED** [review]
Nice work!

**Pull request merged**